cmd/proxy/actions/app_proxy.go

```diff
@@ -62,6 +62,8 @@ func addProxyRoutes(
 
 	handlerOpts := &download.HandlerOpts{Protocol: dp, Logger: l, Engine: proxy}
 	download.RegisterHandlers(app, handlerOpts)
+	// both methods are called until fully removing buffalo
+	download.RegisterHandlersV2(app.Muxer(), handlerOpts)
```

Comment:
Sorry, I'm not completely following. Do you mean the individual routes? If so, then yes, we'd need to remove them from one Register method and add them to the other. But the nice thing about that is we're actively using the routes as we move them rather than switching them all over at once (even if we define them incrementally).